### ISSUE TITLE
Target size of buttons available under Kiota openapi explorer is less than required size 24x24pixel: A11y_VS Code extension for API Center_Generate API Client_Target Size. 

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#104]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #GH_VSCodeextensionforAPICenter_Win32_Apr2024; #DesktopApp; #VS Code Extension for API Center; #A11yMAS; #WCAG2.5.8; #A11yWCAG2.2; #Target size;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
### Pre-requisites:

1. Open system settings.
2. Navigate to Bluetooth & devices->Mouse->Additional mouse settings.
3. Tab to Pointers tab.
4. Tab to browse and hit enter. 
5. Select the 24*24.cur file(please find in attachment).
6. The mouse pointer will change to 24x24 pixel box.
7. You can move the pointer and verify the size of any control.
### Repro Steps:

1. Open Azure portal.
2. Create an API center in azure portal.
3. Open Visual studio Code.
4. Tab till API center extension in the left side panel and press enter.
5. Tab till definition under the Api name which you have created, and press enter.
6. Tab till azure tool manual testing and click on it.
7. Tab till environment and click on it.
8. Tab till open API and press enter.
6. Go to "Generate API Client" using the down arrow and press enter.
7. Kiota openapi explorer window will open.
9. Observe that Target size of buttons available under Kiota openapi explorer is less than required size 24x24pixel.

### Actual Results: 
Target size of buttons available under Kiota openapi explorer is less than required size 24x24pixel
### Expected Results:
Target size of buttons available under Kiota openapi explorer should be minimum 24x24pixel. If the controls have spaced in-between, then those spaces should not intersect each other and target size with spacing should be 24x24 pixel for each one.

### User Impact: 
Users with dexterity limitations and those who have difficulty with fine motor movement find it difficult to accurately activate small targets when there are other targets that are too close. Providing sufficient size, or sufficient spacing between targets, will reduce the likelihood of accidentally activating the wrong control.

### Attachment:
![Bug_104_1](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/21c0565d-5b45-430c-ba9d-314310387ae7)

![Bug104](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/2c72ef1e-81b4-4d21-a6a7-f24898de490b)

**Comment by formulahendry** [comment]
External issue from Microsoft Kiota extension for Visual Studio Code: https://github.com/microsoft/kiota

**Comment by formulahendry** [comment]
This issue is caused by UI of Microsoft Kiota extension, not related to UI of API Center extension.

**Comment by AnKushSingh05** [comment]
### Github Tags:
#FirstParty:Bug212686;#External;#External:VSCode;#BM-VisualStudioCodeClient-Win32-Jan2024;

https://github.com/microsoft/vscode/issues/212686

**Comment by AnKushSingh05** [comment]
Closing this bug as external bug is closed as W=22 but there is enough space between the controls so closing the bug
